apps/src/componentLibrary/simpleDropdown/SimpleDropdown.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
+import moduleStyles from './simpleDropdown.module.scss';
+
+export interface SimpleDropdownProps {
+  /** SimpleDropdown items list */
+  items: {value: string; text: string}[];
+  /** SimpleDropdown selected value */
+  selectedValue?: string;
+  /** SimpleDropdown onChange handler */
+  onChange: (args: React.ChangeEvent<HTMLSelectElement>) => void;
+  /** SimpleDropdown label text */
+  labelText: string;
```

Comment:
Maybe use `dropdownLabelText` since it refers to the dropdown and not an option label.

Reply:
Or just `label` now that the items use `text`?